packages/suite-web/vite.config.ts

```diff
@@ -0,0 +1,163 @@
+import { viteCommonjs } from '@originjs/vite-plugin-commonjs';
+import react from '@vitejs/plugin-react';
+import { execSync } from 'child_process';
+import { readdirSync } from 'fs';
+import { resolve } from 'path';
+import { Plugin, ViteDevServer, defineConfig } from 'vite';
+import { nodePolyfills } from 'vite-plugin-node-polyfills';
+import wasm from 'vite-plugin-wasm';
+
+import { suiteVersion } from '../suite/package.json';
+
+// Use require instead of import for TypeScript files
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { assetPrefix, project } = require('../suite-build/utils/env');
```

Comment:
damn it, I had to do this... Otherviews, the linter was screeming and with `@trezor/suite-build` would allow the vite config to compile, that's mess, but this wworks